pom.xml

```diff
@@ -350,6 +355,11 @@
                 <artifactId>jersey-media-json-jackson</artifactId>
                 <version>${jersey.server.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.glassfish.jersey.core</groupId>
+                <artifactId>jersey-client</artifactId>
+                <version>${jersey.server.version}</version>
+            </dependency>
```

Comment:
We need to remove this from the root pom as well.

Reply:
Done.